Makefile

```diff
@@ -28,3 +28,10 @@ clean-roles:
 
 ansible-galaxy:
 	ansible-galaxy install -r requirements.yml --force
+
+import-gpg-keys:
+	$(foreach var,$(shell cat gpg.recipients | awk -F: '{print $$1}'),gpg --keyserver hkp://pgp.mit.edu --search-keys $(var);)
```

Comment:
We could use macros for `$(shell cat gpg.recipients | awk -F: '{print $$1}')` and `group_vars/all/secure` (below) since we use them more than once and the length makes this quite difficult to read.